### ISSUE TITLE
Youtube watch history

### ISSUE THREAD

**Issue by MarcoDiFrancesco** [StevenBlack/hosts#907]
`0.0.0.0 s.youtube.com  # Blocking this will interfere with video watching history`
I've seen this address is blocked by default. This annoys people that watch youtube from the home page (more or less 60% of youtube users) because the videos keep appearing after you watch them. 
Could you unblock it by default?
Thanks

**Comment by funilrys** [comment]
Hi there, is this a duplicates of #263 #172 ?

**Comment by dnmTX** [comment]
@MarcoDiFrancesco  👉  https://github.com/StevenBlack/hosts/issues/896#issuecomment-456871715

**Comment by MarcoDiFrancesco** [comment]
I understand.
Thanks.
